components/header/NavItem.tsx

```diff
@@ -1,57 +1,111 @@
-            <ul class="flex items-start justify-center gap-6">
-              {children.map((node) => (
-                <li class="p-6">
-                  <a class="hover:underline" href={node.url}>
-                    <span>{node.name}</span>
-                  </a>
-
-                  <ul class="flex flex-col gap-1 mt-4">
-                    {node.children?.map((leaf) => (
-                      <li>
-                        <a class="hover:underline" href={leaf.url}>
-                          <span class="text-xs">{leaf.name}</span>
-                        </a>
-                      </li>
-                    ))}
-                  </ul>
-                </li>
-              ))}
-            </ul>
-          </div>
-        )}
-    </li>
+            {link.label}
+          </a>
+          <li class="flex flex-col">
+            {RenderLinks(link.children, true)}
+          </li>
+        </ul>
+      ))}
+    </>
+  );
+}
+
+function RenderImages(images: Image[] | undefined) {
+  if (!images || images.length === 0) {
+    return null;
+  }
+
+  return (
+    <div class="flex items-start px-2 justify-end w-2/4 flex-wrap">
+      {images.map((imageData) => (
+        <a href={imageData.link} class=" w-1/3 max-w-[180px] lg:max-w-[242px] hover:scale-105 ease-in duration-200">
+          <figure class="flex flex-col items-center w-full px-2 gap-y-[15px] ">
+            <Image
+              src={imageData.image}
+              alt={imageData.alternateName}
+              width={242}
+              height={242}
+              loading="lazy"
+              class="w-full "
+            />
+            <figcaption class="text-center text-base font-semibold leading-tight whitespace-normal">
+              {imageData.label}
+            </figcaption>
+          </figure>
+        </a>
+      ))}
+    </div>
+  );
+}
+
+function NavItem(item: NavItemProps) {
+  const { label, links, images } = item;
+  return (
+    <nav class="group flex items-center cursor-pointer px-0" style={{ height: navbarHeight }}>
+      <div class="relative flex items-center pr-4">
+        <Icon
+          id="Bars3"
+          size={22}
+          strokeWidth={2}
+          fill="none"
+          class="absolute left-0 top-0 transition-opacity opacity-100 group-hover:opacity-0"
+        />
+        <Icon
+          id="Close"
+          size={22}
+          fill="none"
+          class="absolute left-0 top-0 transition-opacity opacity-0 group-hover:opacity-100"
+        />
+        <span class="ml-7 uppercase">
+          {label}
+        </span>
+      </div>
+      <div class="hidden group-hover:flex bg-base-100 z-50 items-start justify-between w-full border-t border-b-2 border-base-200 flex-row-reverse pt-[55px] pb-14 top-full absolute left-0">
+        <div class="flex items-start justify-between flex-row w-full">
+          <ul class="flex flex-1 items-start px-2 w-2/4 justify-start gap-8 flex-wrap xl:gap-16 2xl:gap-24">
+            {RenderLinks(links)}
```

Comment:
A forma correta de chamar um componente, aplique o mesmo no RenderImages
```suggestion
            <RenderLinks links={links} />
```